applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py

```diff
@@ -1,4 +1,19 @@
 import KratosMultiphysics
+import KratosMultiphysics.CompressiblePotentialFlowApplication as CPFApp
+import math
+
+def DotProduct(A,B):
```

Comment:
ouchouchouch ...
this should be implemented in C++, you could check out `kratos/python/add_vector_to_python.cpp`

If you don't want to do this then please use a list comprehension:
`return sum(i[0]*i[1] for i in zip(A, B))`

Reply:
I could not find a dot product implemented in kratos/python/add_vector_top_python.cpp so I used your suggestion.

Reply:
Yes it is not yet exposed but you could add it I guess

But since you are anyway migrating to cpp it is ok